### ISSUE TITLE
bug(@nrwl/node): Error: Unable to resolve @nrwl/node:execute.

### ISSUE THREAD

**Issue by eduardoRoth** [nrwl/nx#9205]
## Current Behavior

When using @nrwl/node@13.8.4 I get the following error:
Error: Unable to resolve @nrwl/node:execute.

## Expected Behavior

It's a regression, error goes away with v13.8.3

## Steps to Reproduce

1. npx create-nx-workspace demo (angular/sass/no)
2. npm install -D @nrwl/nest
3. nx g @nrwl/nest:app api
4. ng serve api

### Failure Logs

```
Error: Unable to resolve @nrwl/node:execute.
Cannot find executor 'execute' in /Users/eduardoroth/trabajo/projects/apps/alliax/recursos-humanos/rh-accesos/node_modules/@nrwl/node/executors.json.
    at Workspaces.readExecutor 
```
```
Error: Unable to load hasher for task "api:serve"
    at getCustomHasher
```

### Environment

```
>  NX   Report complete - copy this into the issue template

Node : 16.14.0
OS   : darwin x64
npm  : 8.5.3

nx : 13.8.4
@nrwl/angular : 13.8.4
@nrwl/cli : 13.8.4
@nrwl/cypress : 13.8.4
@nrwl/detox : undefined
@nrwl/devkit : 13.8.4
@nrwl/eslint-plugin-nx : 13.8.4
@nrwl/express : undefined
@nrwl/jest : 13.8.4
@nrwl/js : 13.8.4
@nrwl/linter : 13.8.4
@nrwl/nest : 13.8.4
@nrwl/next : undefined
@nrwl/node : 13.8.4
@nrwl/nx-cloud : undefined
@nrwl/react : undefined
@nrwl/react-native : undefined
@nrwl/schematics : undefined
@nrwl/storybook : 13.8.4
@nrwl/tao : 13.8.4
@nrwl/web : undefined
@nrwl/workspace : 13.8.4
typescript : 4.5.5
rxjs : 7.5.4
---------------------------------------
Community plugins:
@angular/animations: 13.2.5
@angular/common: 13.2.5
@angular/compiler: 13.2.5
@angular/core: 13.2.5
@angular/forms: 13.2.5
@angular/platform-browser: 13.2.5
@angular/platform-browser-dynamic: 13.2.5
@angular/router: 13.2.5
@ionic/angular: 6.0.10
@ionic/angular-toolkit: 6.0.0
@angular-devkit/build-angular: 13.2.5
@angular/cli: 13.2.5
@angular/compiler-cli: 13.2.5
@angular/language-service: 13.2.5
@nxtend/capacitor: 13.0.0
@nxtend/ionic-angular: 13.0.0
```

**Comment by jpb06** [comment]
See https://github.com/nrwl/nx/issues/9168#issuecomment-1057897421

**Comment by jaypersanchez** [comment]
In my package.json, I have @nrwl/cli and other @nrwl/* at version 13.3.6.  If this can be reproduce with version 13.8.4, what is the solution then?  I am on Windows 10, Node 16.4.0, Npm 8.3.1.

**Comment by AgentEnder** [comment]
This was fixed in 13.8.5, check the thread @jpb06 linked for more info
